### ISSUE TITLE
Muting the Editor also mutes s&box in play mode

### ISSUE THREAD

**Issue by Retroeer** [Facepunch/sbox-issues#5704]
### Describe the bug

If you mute the game in the editor using that little button on the top, then quit the editor, and go into the full play mode, the one where you just launch the sbox.exe, that becomes muted aswell

### To Reproduce

1. mute editor
2. launch sbox in play mode
3. no sound :(

### Expected behavior

Sound should be disabled for the Editor but remain in play mode, thats what the volume slider is for

**Comment by DevulTj** [comment]
Makes sense, all that the Settings/Mute option does is set the snd_mute convar, which is used in the game. Though this probably shouldn't be the case.

**Comment by Retroeer** [comment]
Edited this to fit a broader issue.

**Comment by DoctorGurke** [comment]
Convars are cool but dev should have its own configs from game, per project would be the best
